dev-docs/bidders/aduptech.md

```diff
@@ -2,23 +2,24 @@
 layout: bidder
 title: AdUp Technology
 description: Prebid Bidder Adapter for AdUp Technology
-pbjs: true
 biddercode: aduptech
-media_types: banner
 gdpr_supported: true
+gvl_id: 647
+media_types: banner, native
+pbjs: true
 ---
 
 ### Note:
 
-The AdUp Technology bidding adapter requires setup and approval before beginning.
-For more information visit [www.adup-tech.com](https://www.adup-tech.com/) or contact [info@adup-tech.com](mailto:info@adup-tech.com).
+The AdUp Technology bidding adapter requires correct setup and approval, including an existing publisher account. For more information visit [www.adup-tech.com](https://www.adup-tech.com/en) or contact [info@adup-tech.com](mailto:info@adup-tech.com).
```

Comment:
You could change this to read: 

To use, our bidding adapter requires proper setup,  including an existing publisher account, as well as  approval from AdUp Technology.

Reply:
fixed it, thanks